src/Model/SelfService/CategoryTranslation.php

```diff
@@ -106,4 +108,23 @@ public function setDescription(string $description): self
 
         return $this;
     }
+
+    /**
+     * @return int
+     */
+    public function getCategoryId(): int
+    {
+        return $this->categoryId;
+    }
+
+    /**
+     * @param int $categoryId
+     * @return CategoryTranslation
+     */
+    public function setCategoryId(int $categoryId): self
```

Comment:
most of these docblocks are redundant? they just replicate the function params / type hint.

Reply:
let's ignore this, i see it follows the format of the rest of the file :shrug: